src/python/pants/core/util_rules/environments.py

```diff
@@ -201,6 +201,15 @@ class DockerImageField(StringField):
         """
     )
 
+class DockerBindMountsField(StringSequenceField):
+    alias = "bind_mounts"
```

Comment:
Maybe just call this `mounts`? The Docker documentation just calls them mounts (and the corresponding `docker container run` option is `--mount`.)

Reply:
(At least for the Docker CLI documentation.)